mirrord/cli/src/config.rs

```diff
@@ -250,3 +250,11 @@ pub(super) struct ExtensionExecArgs {
     #[arg(short = 't')]
     pub target: Option<String>,
 }
+
+#[derive(Args, Debug)]
+pub(super) struct InternalProxyArgs {
+    /// Launch timeout until we get first connection.
+    /// If layer doesn't connect in this time, we timeout and exit.
+    #[arg(short = 't', default_value_t = 2)]
```

Comment:
Probably long should still be here
```suggestion
    #[arg(short = 't', long, default_value_t = 2)]
```

Reply:
This isn't used by the outside so I don't see a reason to add it..